docker-compose.yml

```diff
@@ -15,18 +15,18 @@ services:
       - "7777:7777"
     environment:
       - RUN_DEV_SERVER=True
-      - PORT=7777
-      - HOST=0.0.0.0
       - DEBUG=True
+      - PROTOCOL=https
```

Comment:
I suggest a default of HTTPS, and we explicitly override that in the docker-compose.yml file with an env var.  Secure by default.